src/pages/ProfilePage.tsx

```diff
@@ -267,38 +250,35 @@ function ProfilePage(props) {
                         icon={Illustrations.ToddBehindCloud}
                         iconWidth={variables.modalTopIconWidth}
                         iconHeight={variables.modalTopIconHeight}
-                        title={props.translate('notFound.notHere')}
+                        title={translate('notFound.notHere')}
                         shouldShowLink
-                        link={props.translate('notFound.goBackHome')}
                     />
                 )}
             </View>
         </ScreenWrapper>
     );
 }
 
-ProfilePage.propTypes = propTypes;
-ProfilePage.defaultProps = defaultProps;
 ProfilePage.displayName = 'ProfilePage';
 
-export default compose(
-    withLocalize,
-    withOnyx({
-        personalDetails: {
-            key: ONYXKEYS.PERSONAL_DETAILS_LIST,
-        },
-        session: {
-            key: ONYXKEYS.SESSION,
-        },
-        report: {
-            key: ({route, session}) => {
-                const accountID = Number(lodashGet(route.params, 'accountID', 0));
-                const reportID = lodashGet(ReportUtils.getChatByParticipants([accountID]), 'reportID', '');
-                if ((session && Number(session.accountID) === accountID) || Session.isAnonymousUser() || !reportID) {
-                    return null;
-                }
-                return `${ONYXKEYS.COLLECTION.REPORT}${reportID}`;
-            },
+export default withOnyx<ProfilePageProps, ProfilePageOnyxProps>({
+    personalDetails: {
+        key: ONYXKEYS.PERSONAL_DETAILS_LIST,
+    },
+    session: {
+        key: ONYXKEYS.SESSION,
+    },
+    report: {
+        key: ({route, session}) => {
+            const accountID = Number(route.params?.accountID ?? 0);
+            const reportID = ReportUtils.getChatByParticipants([accountID])?.reportID ?? '';
+
+            // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+            if ((session && Number(session.accountID) === accountID) || SessionActions.isAnonymousUser() || !reportID) {
```

Comment:
```suggestion
            if ((!!session && Number(session.accountID) === accountID) || SessionActions.isAnonymousUser() || !reportID) {
```

Reply:
I think it better to keep it as it was before since there is no difference in the result of this expression

Reply:
Same

Reply:
updated